examples/demos/GroupedColumnsPanel.js

```diff
@@ -0,0 +1,87 @@
+import React, { Component } from 'react';
```

Comment:
Do we plan on using these js files?

Reply:
Like I mentioned in the previous comment, this was used in the Group example but we cannot tsfy them at the moment. However, it's working from my old PR changes.